### ISSUE TITLE
Illustrations/Pictures of any kind?

### ISSUE THREAD

**Issue by NathanDreamweaver** [NicoAICP/PTPE#2]
For someone who is not the absolute most skilled at setting up stuff like this, is there anything a bit easier to follow along with?

**Comment by NicoAICP** [comment]
What exactly do you mean?

Like Pictures as in building the hardware for PTPE?

**Comment by NathanDreamweaver** [comment]
Something similar to what was on the KAOS repo

**Comment by NicoAICP** [comment]
Yeah that is planned as soon as PTPE is useable (it currently does not fully emulate skylanders and by this is not usuable)

**Comment by Dune808** [comment]
> Yeah that is planned as soon as PTPE is useable (it currently does not fully emulate skylanders and by this is not usuable)

Sorry to bother, is there any update on this?

**Comment by NicoAICP** [comment]
I am currently recording a video for building it, and also writing a guide with pictures. Lazr has made a working PCB tho, if you want to use one of those.
https://github.com/Lazr1026/PTPE_PCB
